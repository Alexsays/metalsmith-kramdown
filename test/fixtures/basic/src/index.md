
# A Kramdown Post

With some "amazing", _riveting_, **coooonnnntent**.