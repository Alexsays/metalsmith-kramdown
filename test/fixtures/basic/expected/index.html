<h1 id="a-kramdown-post">A Kramdown Post</h1>
<p>With some “amazing”, <em>riveting</em>, <strong>coooonnnntent</strong>.</p>
